static/css/less/font-families.less

```
@@ -1,21 +1,8 @@
-@arial_sans_serif: "Arial","Helvetica",sans-serif;
-
-@georgia_serif-1: Georgia, "Palatino Linotype", "Book Antiqua", Palatino, serif;
-@georgia_serif-2: "Georgia","Times New Roman",serif;
-
-@news_gothic_sans_serif: "News Gothic MT","Arial Rounded MT",Geneva,Helvetica,sans-serif;
-@news_gothic_sans_serif-2: "News Gothic MT","Trebuchet MS",Geneva,Helvetica,sans-serif;
-
-@lucida_sans_serif-1: "Lucida Grande", Verdana, Geneva, Helvetica, Arial, sans-serif;
-@lucida_sans_serif-2: "Lucida Grande",Verdana,Geneva,Helvetica,sans-serif;
-@lucida_sans_serif-3: "Lucida Grande","Arial","Helvetica",sans-serif;
-@lucida_sans_serif-4: "Lucida Grande", "Trebuchet MS", Geneva, Helvetica, sans-serif;
-@lucida_sans_serif-5: "Lucida Grande", "Trebuchet MS", Geneva, Helvetica, Arial, sans-serif;
-@lucida_sans_serif-6: "Lucida Grande", Verdana, Geneva, Helvetica, sans-serif;
-@lucida_sans_serif-7: "Lucida Grande","Arial",sans-serif;
-
-@lucida_console_monospace: "Lucida Console","Courier New", monospace;
-
+@arial_sans_serif: "Arial", "Helvetica", sans-serif;
+@lucida_sans_serif: -apple-system, BlinkMacSystemFont, "Segoe UI", "Noto Sans", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji";
```

This is triggering a lint error:

```
 2:148  ✖  Expected line length to be no more than 100          max-line-length
           characters
```

```suggestion
// stylelint-disable-next-line max-line-length
@lucida_sans_serif: -apple-system, BlinkMacSystemFont, "Segoe UI", "Noto Sans", Helvetica, Arial, sans-serif, "Apple Color Emoji", "Segoe UI Emoji";
```